apps/condo/domains/organization/utils/testSchema/index.js

```diff
@@ -21,6 +21,39 @@ const OrganizationEmployee = generateGQLTestUtils(OrganizationEmployeeGQL)
 const OrganizationLink = generateGQLTestUtils(OrganizationLinkGQL)
 /* AUTOGENERATE MARKER <CONST> */
 
+async function makeClientWithOrganization (extraAttrs = {}) {
```

Comment:
we already have this util!

Reply:
please, look at existed functions before adding a new one! `makeClientWithRegisteredOrganization`